core/indigo-core/molecule/monomers_basic_templates.h

```diff
@@ -113,15 +113,15 @@ namespace indigo
                                          "                \"attachmentAtom\": 0,\n"
```

Comment:
May be better use raw string for multiline strings?
Something like
char *string =  R""(multi
    "multi": "line"
 string)"";

Reply:
this header will be autogenerated. here is a ticket: https://github.com/epam/Indigo/issues/1429